test/common/upstream/load_balancer_fuzz_base.cc

```diff
@@ -0,0 +1,193 @@
+#include "test/common/upstream/load_balancer_fuzz_base.h"
+
+#include "test/common/upstream/utility.h"
+
+namespace Envoy {
+namespace Upstream {
+
+namespace {
+
+constexpr uint32_t MaxNumHostsPerPriorityLevel = 256;
+
+} // namespace
+
+void LoadBalancerFuzzBase::initializeASingleHostSet(
+    const test::common::upstream::SetupPriorityLevel& setup_priority_level,
+    const uint8_t priority_level) {
+  const uint32_t num_hosts_in_priority_level = setup_priority_level.num_hosts_in_priority_level();
+  ENVOY_LOG_MISC(trace, "Will attempt to initialize host set {} with {} hosts.", priority_level,
+                 num_hosts_in_priority_level);
+  MockHostSet& host_set = *priority_set_.getMockHostSet(priority_level);
+  uint32_t hosts_made = 0;
+  // Cap each host set at 256 hosts for efficiency - Leave port clause in for future changes
+  while (hosts_made < std::min(num_hosts_in_priority_level, MaxNumHostsPerPriorityLevel) &&
+         port_ < 65535) {
+    host_set.hosts_.push_back(makeTestHost(info_, "tcp://127.0.0.1:" + std::to_string(port_)));
+    ++port_;
+    ++hosts_made;
+  }
+
+  Fuzz::ProperSubsetSelector subset_selector(setup_priority_level.random_bytestring());
+
+  const std::vector<std::vector<uint8_t>> localities = subset_selector.constructSubsets(
+      {setup_priority_level.num_hosts_locality_one(), setup_priority_level.num_hosts_locality_two(),
+       setup_priority_level.num_hosts_locality_three()},
+      std::min(num_hosts_in_priority_level, MaxNumHostsPerPriorityLevel));
+
+  HostVector locality_one = {};
+  HostVector locality_two = {};
+  HostVector locality_three = {};
```

Comment:
are these vars every used?

Reply:
Yeah, Adi mentioned to make it an array for the fixed size property :)!

Reply:
Here's what I'll do - the main reason I declared these with var names is so someone reading can quickly understand, I'll switch my logs to locality_one instead of localities[0]

Reply:
I also added a comment.